packages/suite/src/actions/wallet/blockchainActions.ts

```diff
@@ -72,25 +76,56 @@ export const preloadFeeInfo = () => async (dispatch: Dispatch) => {
 export const updateFeeInfo = (symbol: string) => async (dispatch: Dispatch, getState: GetState) => {
     const network = getNetwork(symbol.toLowerCase());
     if (!network) return;
-
     const blockchainInfo = getState().wallet.blockchain[network.symbol];
     const feeInfo = getState().wallet.fees[network.symbol];
+
     if (feeInfo.blockHeight > 0 && blockchainInfo.blockHeight - feeInfo.blockHeight < 10) return;
 
-    const result = await TrezorConnect.blockchainEstimateFee({
-        coin: network.symbol,
+    let payload: {
```

Comment:
import { BlockchainEstimateFee } from 'trezor-connect';

you should use this type